docs/source/notebooks/Bayes_factor.ipynb

```diff
@@ -346,7 +346,7 @@
    "source": [
     "As we can see from the previous cell, SMC gives essentially the same answer as the analytical calculation! \n",
     "\n",
-    "The advantage of using SMC is that we can use it to compute the _marginal likelihood_ for a wider range of models, given that we do not need an analytical expression for the _marginal likelihood_. The cost we pay is that the computation is more expensive with SMC. We should also take into account that for more complex models we should increase the number of `n_steps` and `n_chains` for a more accurate estimation of the _marginal likelihood_."
+    "The advantage of using SMC is that we can use it to compute the _marginal likelihood_ for a wider range of models as we a closed-form expresion is no longer needed. The cost we pay for this flexibility is a more expensive computation. We should take into account that for more complex models a more accurate estimation of the _marginal likelihood_ will most likely need a larger number of `draws`. Additionally, a larger number of `n_steps` may help, specially if after stage 1 we notice that SMC uses a number of steps that are close to `n_steps`, i.e. SMC is having trouble to automatically reduce this number. "
```

Comment:
Typo in "...for a wider range of models as we a closed-form expresion..."
Delete 'we' here.